src/m365/spo/commands/site/site-recyclebinitem-restore.spec.ts

```diff
@@ -117,7 +138,65 @@ describe(commands.SITE_RECYCLEBINITEM_RESTORE, () => {
     assert.equal(result, undefined);
   });
 
-  it('catches error when restores all items from recyclebin', async () => {
+  it('restores all items from the first-stage recycle bin', async () => {
+    const postRequestSpy = sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === 'https://contoso.sharepoint.com/_api/web/RecycleBin/RestoreAll') {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: {
+        siteUrl: 'https://contoso.sharepoint.com',
+        allPrimary: true
+      }
+    });
+
+    assert(postRequestSpy.calledOnce, 'postRequestSpy.calledOnce');
```

Comment:
Let's remove the 2nd parameter from the assert function here, and in other tests as well.